assets/css/style.css

```diff
@@ -0,0 +1,58 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-weight: 300;
+    font-size: 1.05rem;
+    line-height: 1.6;
+    font-family: 'Poppins', sans-serif;
+}
+
+header {
+    background-color: #5C5B5B;
+    padding: 0px 15px;
+}
+
+header img { 
+    height: 200px;
+    width: 200px;
+}
+
+header h1 {
+    flex: auto;
+    font-size: 500%;
+    line-height: 25%;
+    color: #ffffff;
+    margin-bottom: 20%;
+    text-align: center;
+    font-family:'Amatic SC';
+    font-weight: 500%;
+}
```

Comment:
If you look at the developer console and target the h1 it shows that the font-weight: 500% is invalid so you can remove that.